jetbrains-core-gui/tst/software/aws/toolkits/jetbrains/settings/SetSamCli.kt

```diff
@@ -22,13 +22,9 @@ import software.aws.toolkits.jetbrains.services.lambda.sam.SamExecutable
 class SetSamCli : GuiTestCase() {
     @Test
     fun setSamCli() {
-        val instance = ExecutableManager.getInstance().getExecutable<SamExecutable>().thenApply {
-            if (it is ExecutableInstance.Executable) {
-                it
-            } else {
-                null
-            }
-        }.toCompletableFuture().join()
+        val instance = ExecutableManager.getInstance().getExecutable<SamExecutable>().toCompletableFuture().join().takeIf {
```

Comment:
`castSafelyTo<>()` in `com.intellij.util` can clean this up

Reply:
Fixed